trade-finance-manager-ui/server/types/express-session.d.ts

```diff
@@ -0,0 +1,8 @@
+import { TfmSessionUser } from './tfm-session-user';
+
+declare module 'express-session' {
+  interface SessionData {
+    user: TfmSessionUser;
+    userToken: string;
+  }
+}
```

Comment:
This allows our custom `user` and `userToken` properties to be available on the `req.session` object in route handlers. See [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-session/index.d.ts#L199-L211) for details.